it/auth/IndividualLossesApiAuthMainAgentsOnlyISpec.scala

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package auth
+
+import api.services.DownstreamStub
+import play.api.http.Status
+import play.api.libs.json.{JsValue, Json, OWrites, Writes}
+import play.api.libs.ws.{WSRequest, WSResponse}
+import v4.models.domain.lossClaim.TypeOfClaim
+import v5.lossClaims.amendOrder.def1.model.request.Claim
+
+class IndividualLossesApiAuthMainAgentsOnlyISpec extends AuthMainAgentsOnlyISpec {
+
+  val callingApiVersion = "5.0"
+
+  val supportingAgentsNotAllowedEndpoint = "amend-loss-claims-order"
+
+  val mtdUrl = s"/$nino/loss-claims/order/2023-24"
+
+  def sendMtdRequest(request: WSRequest): WSResponse = await(request.put(requestJson()))
+
+  val downstreamUri = s"/income-tax/claims-for-relief/preferences/23-24/$nino"
+
+  override val downstreamHttpMethod: DownstreamStub.HTTPMethod = DownstreamStub.PUT
+
+  val maybeDownstreamResponseJson: Option[JsValue] = None
+
+  override protected val downstreamSuccessStatus: Int = Status.NO_CONTENT
+
+  def requestJson(): JsValue = {
+    val claim1: Claim                 = Claim("1234567890ABEF1", 1)
+    val claim2: Claim                 = Claim("1234567890ABCDE", 2)
+    val claim3: Claim                 = Claim("1234567890ABDE0", 3)
+    val claimSeq: Seq[Claim]          = Seq(claim2, claim1, claim3)
```

Comment:
Can this just be "claims"?  We try to avoid including the type in the val name, except for some v.specific cases.

Also just while you're there, "= Seq(..)" could be "= List(..)" - there's a bunch of reasons in our coding WoW page.